canonical/ubuntu.com
# WD-13653 Update legal agreement

## Done

- Updated legal confidentiality agreement

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Most of the changes is just the formatting
- [Agreement](https://docs.google.com/document/d/0B8zvYdcwzBYOa294Zk1QTjBOZE0/edit?resourcekey=0-uV9FjHSjn2CZcD2mFCjRIg)
- [Demo](https://ubuntu-com-14141.demos.haus/legal/confidentiality-agreement)
- Visit the demo and please verify if the agreement on the page is the updated one.

## Issue / Card

Fixes # [WD-12653](https://warthogs.atlassian.net/browse/WD-13653)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-12653]: https://warthogs.atlassian.net/browse/WD-12653?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
